### PR TITLE
Feature/new platforms

### PR DESCRIPTION
**Pull request by StephenHodgson**
# XRTK - Mixed Reality Toolkit Pull Request

## Overview

- Added https://github.com/XRTK/ARCore
- Added https://github.com/XRTK/Etee
- Added https://github.com/XRTK/Examples
- Added https://github.com/XRTK/Lenovo
- Added https://github.com/XRTK/Nreal
- Added https://github.com/XRTK/SteamVR
- Added https://github.com/XRTK/Ultraleap
- Updated build pipeline to build and validate android builds

## Submodule Changes

- [x] [ARCore](https://github.com/XRTK/ARCore/pull/1)
- [x] [Etee](https://github.com/XRTK/Etee/pull/1)
- [x] [Examples](https://github.com/XRTK/Examples/pull/7)
- [x] [Lenovo](https://github.com/XRTK/Lenovo/pull/1)
- [x] [Nreal](https://github.com/XRTK/Nreal/pull/2)
- [x] [SteamVR (OpenVR)](https://github.com/XRTK/SteamVR/pull/1)
- [x] [Ultraleap](https://github.com/XRTK/UltraLeap/pull/3)

### CODE REVIEW

**Comment by StephenHodgson** [comment]
Remember to review the submodules and let them get merged to dev before approving this PR

File: XRTK-Core/Assets/XRTK.Examples [hunk 1]
```diff
@@ -0,0 +1 @@
+C:/Dev/XRTK/Core/Submodules/Examples/XRTK.Examples/Assets/XRTK.Examples
```
Comment (StephenHodgson) [inline XRTK-Core/Assets/XRTK.Examples:1]: Needs to be removed and ignored

**Comment by SimonDarksideJ** [comment]
etee showing ticked in the list but has not been added as a submodule @StephenHodgson ?

**Comment by StephenHodgson** [comment]
I have not added it yet. Still working on the ARCore setup and Template Gen Tutorial

**Comment by SimonDarksideJ** [comment]
#598 submitted to add etee module and fix gitnore file for new platforms

**Comment by StephenHodgson** [comment]
Thanks mate! approved

**Pull request merged**